### DISCUSSION TITLE
Would it make sense for PowerOnAndConnect to generate a WOL packet if needed?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [WesSouza/lgtv-ip-control#121] [Ideas]
Would it make sense for PowerOnAndConnect to generate a WOL packet if needed?

**Comment by WesSouza** [comment]
It already does that:
https://github.com/WesSouza/lgtv-ip-control/blob/75026f700782867a0a2c6e1394248237b1d42699/packages/lgtv-ip-control/src/classes/LGTV.ts#L124

Power on can only be done via wake on lan.

**Comment by Jibun-no-Kage** [comment]
Right I should checked the code!
